src/client/sound.cpp

```diff
@@ -101,7 +101,11 @@ void sound_volume_control(ISoundManager *sound_mgr, bool is_window_active)
 {
 	bool mute_sound = g_settings->getBool("mute_sound");
 	if (mute_sound) {
+#if defined(__APPLE__)
+		sound_mgr->setListenerGain(0.001f);
+#else
 		sound_mgr->setListenerGain(0.0f);
+#endif
```

Comment:
What if someone explicitly sets `sound_volume = 0`?

Reply:
Moved to `OpenALSoundManager::setListenerGain`